sodetlib/stream.py

```diff
@@ -207,6 +208,11 @@ def stream_g3_on(S, make_freq_mask=False, emulator=False, tag=None,
     S.set_downsample_factor(downsample_factor)
     S.set_filter_disable(int(filter_disable))
 
+    # ensure fixed tones are on
+    for band in S.bands:
+        if "fixed_tones" in cfg.dev.bands[band]:
+            uxm_setup.turn_on_fixed_tones(S, cfg, band)
```

Comment:
I'm not sure this will work since you need to run tracking_setup after turning on fixed tones.

Reply:
I think we should just figure out how to enable fixed tones without tracking setup... I imagine there's just a register or two that needs to be set.

Reply:
I had looked through `tracking_setup` and it wasn't clear to me what would be missing, but in any case I will have to go through all of these steps with testing and I should find out then.

Reply:
I added a `turn_on_fixed_tones` call to `relock_tracking_setup` so that if it disables these channels they should be restored

Reply:
I think we still need an additional `enable_fixed_tones` cfg entry which allows us to disable fixed tones without removing the fixed-tone dictionary.